### PR TITLE
Add windows cr

### PR DESCRIPTION
**Pull request by amirbilu**
Hi @instance-id, highly appreciate the PR and sorry for the late reply.

See my adjustments here and let me know if you're okay with it.

I've tried to run the `dl_binaries.ps1`  script on a colleague Windows machine and got this error. Any idea? unfortunately I don't have time to investigate this:
```powershell
Expand-Archive : A parameter cannot be found that matches parameter name 'outputpath'.
At C:\Users\tzufs\AppData\Local\nvim-data\site\pack\packer\start\tabnine-nvim\dl_binaries.ps1:27 char:55
+ ... expand-archive -path "binaries/$path/TabNine.zip" -outputpath "binari ...
+                                                       ~~~~~~~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Expand-Archive], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Expand-Archive
```

image

### CODE REVIEW

**Comment by MostHated** [comment]
I looked that up on the powershell issues page, and it appears to be a common error, as the 'Expand-Archive' command for Window Powershell (v5.1) and possibly also Powershell Core (v6) use -destinationpath (unless you have the pscx module installed, which I did, so -outputpath worked for me in both Powershell 5.1 and 7.x), but newer Powershell switched to using the -outputpath. I can make a modification to it to check the used version and use the proper output command.

**Comment by instance-id** [comment]
I looked that up on the powershell issues page, and it appears to be a common error, as the 'Expand-Archive' command for Window Powershell (v5.1) and possibly also Powershell Core (v6) use -destinationpath (unless you have the pscx module installed, which I did, so -outputpath worked for me in both Powershell 5.1 and 7.x), but newer Powershell switched to using the -outputpath. I can make a modification to it to check the used version and use the proper output command.

**Comment by amirbilu** [comment]
@instance-id sounds great. Let me know of when this is done

**Comment by MostHated** [comment]
I added the check and pushed it to my initial branch. I hate to say, but I am not even sure how I would, or if I should have pushed it in relation to what you did here. I have not had to get far enough into the weeds with git to deal with what looks like a branch off a pr's branch? If I did push it correctly, it should be good to test the powershell script again.

**Comment by instance-id** [comment]
I added the check and pushed it to my initial branch. I hate to say, but I am not even sure how I would, or if I should have pushed it in relation to what you did here. I have not had to get far enough into the weeds with git to deal with what looks like a branch off a pr's branch? If I did push it correctly, it should be good to test the powershell script again.

**Comment by amirbilu** [comment]
This is okay :)
I'll check your PR on the Windows machine at the office tomorrow. Much appreciated!

**Comment by amirbilu** [comment]
okay.. I approve this is working now. if you can please merge this PR and then I'll merge yours.

File: README.md
Comment (instance-id) [inline README.md:1]: Was this intended to change `pwsh.exe -file .\\dl_binaries.ps1` to  `pwsh.exe -file .\\dl_binaries.ps`?

Comment (amirbilu) [reply README.md:1]: oh no! sorry about this

Comment (amirbilu) [reply README.md:1]: fixed

**Review by instance-id — APPROVED** [review]

**Pull request merged**